manage.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
+"""Django's command-line utility for administrative tasks."""
+import os
+import sys
+
+
+def main():
+    """Invoke main function."""
+    os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'mysite.settings')
+    try:
+        # pylint: disable=C0415
```

Comment:
You can also use more descriptive rule identifiers rather than just numeric IDs. It's easier to understand what the rule implements w/o having to google it:
```suggestion
        # pylint: disable=import-outside-toplevel
```

Reply:
Still, I'm not sure why this message appears here. Maybe it could go away by just installing django rather than trying to ignore things.